charts/ztka/values.yaml

```diff
@@ -296,6 +296,16 @@ kratos:
               oidc:
                 hooks:
                   - hook: session
+          login:
+            ui_url: http://dashboard:3000/login
```

Comment:
you may want to check if the port if configurable, then hardcoding it might not work.

Reply:
Completed